src/vmm/src/arch/aarch64/fdt.rs

```diff
@@ -258,6 +275,22 @@ fn create_chosen_node(
     Ok(())
 }
 
+fn create_vmgenid_node(fdt: &mut FdtWriter, vmgenid: &Option<VmGenId>) -> Result<(), FdtError> {
+    if let Some(vmgenid_info) = vmgenid {
+        let compatible = "microsoft,vmgenid";
+
+        let vmgenid = fdt.begin_node("vmgenid")?;
+        fdt.property_string("compatible", compatible)?;
```

Comment:
just write it in one line:
`fdt.property_string("compatible", compatible)?;` -> `fdt.property_string("compatible", "microsoft,vmgenid")?;`